page-job-listing.php

```diff
@@ -0,0 +1,443 @@
+$dropdown_html_region = wp_dropdown_categories(
+    array(
+        'name' => 'job_region',
+        'id' => 'job-filter-region',
+        'class' => 'govuk-select',
+        'taxonomy' => 'job_region',
+        'show_option_all' => 'Select option',
+        'orderby' => 'name',
+        'echo' => 0,
+        'hide_if_empty' => 1,
+        'selected' => $selected_job_region_id,
+        'exclude' => $national_term_ID
+    )
+);
+
+if ($selected_job_max_salary_id != "0" && $selected_job_max_salary_id < $selected_job_min_salary_id ) {
+    $salaryError = true;
+    $salaryErrorClass = "govuk-select--error";
+    $salaryErrorMin = "Minimum salary cannot be higher than maximum salary.";
+    $salaryErrorMax = "Maximum salary cannot be lower than minimum salary.";
+}
+
+$selected0 = $selected20000 = $selected30000 = $selected40000 = $selected50000 = $selected60000 = $selected70000 = $selected80000 = $selected90000 = $selected100000 = "";
+$salarySelected = "selected$selected_job_min_salary_id";
+$$salarySelected = "selected";
+$dropdown_html_min_salary =
+"
+<select class='govuk-select $salaryErrorClass' name='min_salary' id='job-filter-min-salary'>
+    <option value='0' $selected0>No minimum</option>
+    <option value='20000' $selected20000>£20,000</option>
+    <option value='30000' $selected30000>£30,000</option>
+    <option value='40000' $selected40000>£40,000</option>
+    <option value='50000' $selected50000>£50,000</option>
+    <option value='60000' $selected60000>£60,000</option>
+    <option value='70000' $selected70000>£70,000</option>
+    <option value='80000' $selected80000>£80,000</option>
+    <option value='90000' $selected90000>£90,000</option>
+    <option value='100000' $selected100000>£100,000</option>
+</select>
+";
+
+$selected0 = $selected20000 = $selected30000 = $selected40000 = $selected50000 = $selected60000 = $selected70000 = $selected80000 = $selected90000 = $selected100000 = "";
+$salarySelected = "selected$selected_job_max_salary_id";
+$$salarySelected = "selected";
+$dropdown_html_max_salary =
+"
+<select class='govuk-select $salaryErrorClass' name='max_salary' id='job-filter-max-salary'>
+    <option value='0' $selected0>No maximum</option>
+    <option value='20000' $selected20000>£20,000</option>
+    <option value='30000' $selected30000>£30,000</option>
+    <option value='40000' $selected40000>£40,000</option>
+    <option value='50000' $selected50000>£50,000</option>
+    <option value='60000' $selected60000>£60,000</option>
+    <option value='70000' $selected70000>£70,000</option>
+    <option value='80000' $selected80000>£80,000</option>
+    <option value='90000' $selected90000>£90,000</option>
+    <option value='100000' $selected100000>£100,000</option>
+</select>
+";
+
+$selected12 = $selected25 = $selected50 = $selected100 = "";
+$sizeSelected = "selected$page_size";
+$$sizeSelected = "selected";
+$dropdown_html_page_count =
+"
+<select class='govuk-select' name='page_size' id='job-filter-page-size'>
```

Comment:
are we keeping this?

Reply:
Feedback from Anna is that we should have a "show all" - I have added this and confirmed wording with her.  

Assuming this is being kept, but Leigh, Scott might yet ask for it to be removed.

Reply:
Decided to remove page size options as they haven't been user tested.